wagtail/admin/static_src/wagtailadmin/js/core.js

```diff
@@ -115,7 +115,7 @@ $(function() {
     function initLogo() {
         var sensitivity = 8; // the amount of times the user must stroke the wagtail to trigger the animation
 
-        var $logoContainer = $('.wagtail-logo-container__desktop');
+        var $logoContainer = $('.animated-logo-container');
```

Comment:
```suggestion
        var $logoContainer = $('[data-animated-logo-container]');
```

Since this class is only used in JS, I would suggest we refactor this to a data- attribute.